Scripts/Core/EditorVR.cs

```diff
@@ -96,6 +96,7 @@ static void ResetPreferences()
 
 		void Awake()
 		{
+			ObjectUtils.hideFlags = defaultHideFlags;
```

Comment:
He's saying it _already_ works on re-compile.  What is missing is that restarting EditorVR (closing and re-opening the window) wouldn't get the updated hide-flags.